rqt_py_common/CMakeLists.txt

```diff
@@ -2,9 +2,25 @@ cmake_minimum_required(VERSION 2.8.3)
 project(rqt_py_common)
 # Load catkin and all dependencies required for this package
 find_package(catkin REQUIRED)
-catkin_package()
 catkin_python_setup()
 
+if(CATKIN_ENABLE_TESTING)
+  find_package(genmsg)
```

Comment:
Since these packages are needed you should pass `REQUIRED`.

Same one line below.